firstradeAPI.py

```diff
@@ -41,10 +41,11 @@ def firstrade_init(FIRSTRADE_EXTERNAL=None):
             print(f"The following Firstrade accounts were found: {account_list}")
             print("Logged in to Firstrade!")
             firstrade_obj.set_logged_in_object(name, firstrade)
-            for i, account in enumerate(account_list):
-                firstrade_obj.set_account_number(name, account)
+            for i, entry in enumerate(account_info.all_accounts):
+                account = list(entry.keys())
```

Comment:
Here I would set account equal to the dict value that is needed, rather than converting it to a list and pulling the first item. That way using `account[0]` shouldn't be needed.

Reply:
all_accounts is a list of dictionaries. as I have it in the API right now. While I agree that would be optimal I need to redo the list of dictionaries in the api I think

Reply:
since this is a list of dictionaries it will only return one key right now, but .keys() returns a list